src/coreclr/src/System.Private.CoreLib/src/System/Text/StringBuilder.CoreCLR.cs

```diff
@@ -104,58 +105,46 @@ internal unsafe void ReplaceBufferAnsiInternal(sbyte* newBuffer, int newLength)
         /// Copies the contents of this builder to the specified buffer.
         /// </summary>
         /// <param name="dest">The destination buffer.</param>
-        /// <param name="len">The number of bytes in the destination buffer.</param>
-        internal unsafe void InternalCopy(IntPtr dest, int len)
+        /// <param name="charLen">The number of chars in the destination buffer.</param>
+        internal unsafe void InternalCopy(IntPtr dest, int charLen)
```

Comment:
```suggestion
        internal unsafe void InternalCopy(IntPtr dest, int charLen) =>
            CopyTo(0, new Span<char>((char*)dest, charLen), charLen);
```
and delete the rest.